apis/apps/v1alpha1/cluster_types.go

```diff
@@ -706,6 +716,11 @@ type ClusterComponentSpec struct {
 	// +optional
 	VolumeClaimTemplates []ClusterComponentVolumeClaimTemplate `json:"volumeClaimTemplates,omitempty" patchStrategy:"merge,retainKeys" patchMergeKey:"name"`
 
+	// List of volumes to override.
+	//
+	// +optional
+	Volumes []corev1.Volume `json:"volumes,omitempty"`
```

Comment:
I think "user defined" env and volumes is way out of scope of a (DB) Cluster model. //cc @weicao  @nayutah

Reply:
User can provide keys or passwords by mounting secret for sshd service running inside the pod